src/composables/maxAmount.ts

```diff
@@ -82,6 +82,9 @@ export function useMaxAmount({ store, formModel }: MaxAmountOptions) {
         selectedAssetDecimals.value = val.selectedAsset.decimals!;
       }
 
+      const numericAmount = (val.amount && val.amount > 0) ? val.amount : 0;
```

Comment:
```suggestion
      const numericAmount = (val?.amount > 0) ? val.amount : 0;
```

Reply:
Nope, you can't check if null is bigger than a number.